mne/viz/topo.py

```diff
@@ -632,12 +633,15 @@ def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         See matplotlib documentation for more details.
     axes : instance of matplotlib Axes | None
         Axes to plot into. If None, axes will be created.
-    show : bool
-        Show figure if True.
     noise_cov : instance of Covariance | str | None
         Noise covariance used to whiten the data while plotting.
         Whitened data channels names are shown in italic.
         Can be a string to load a covariance from disk.
+    exclude : list of str | 'bads'
+        Channels names to exclude from being shown. If 'bads', the
+        bad channels are excluded. If None, exclude is set to 'bads'.
```

Comment:
if it can be `None`, you should add None also in L640

Reply:
actually I just tried and `None` does not work

Reply:
`None` in this case would be the lack of any input param. So basically indicating that exclude would default to 'bads' if no input is received.

Reply:
Should I update the docstring to refernce it as `By default, it's set to 'bads'` ?

Reply:
yep, update the docstring!